### DISCUSSION TITLE
Award check (search QSO for list of the callsigns for the date range)

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2003] [Ideas / Feature Requests]
It would be good to have the possibility to search QSO with the callsigns from the list (comma separated, e.t.c.) for a defined date range. This would be a good automation to check awards like
VRK80 AWARD -- https://www.sk5aa.se/?page_id=8194
HNARC 2023 award -- https://hnarc.gr/hnarc-award-2023
e.t.c..

**Comment by magicbug** [comment]
Currently no it would require extra logic which I don't think most users want currently.

If course you maybe able todo this with raw SQL straight from the database.

**Reply by R1BLH** [reply]
> Currently no it would require extra logic which I don't think most users want currently.
> 
> If course you maybe able todo this with raw SQL straight from the database.

Yes, I see. Already had the idea of making a Win application to query either from a database or by searching in an ADIF file.
